lib/MusicBrainz/Server/Data/LinkType.pm

```diff
@@ -114,14 +114,27 @@ sub load
 
 sub get_tree
 {
-    my ($self, $type0, $type1) = @_;
+    my ($self, $type0, $type1, $get_deprecated_and_empty) = @_;
 
     my %id_to_obj;
     my @objs;
+    my $extra_condition = '';
+
+    unless ($get_deprecated_and_empty) {
```

Comment:
This might make things a little clearer:

```suggestion
    my ($self, $type0, $type1, %opts) = @_;

    my %id_to_obj;
    my @objs;
    my $extra_condition = '';

    unless ($opts{get_deprecated_and_empty}) {
```
then instead of doing `get_tree($type0, $type1, 1)` you would do `get_tree($type0, $type1, get_deprecated_and_empty => 1)`, which is more self-documenting.

Reply:
Thanks, I wanted that but always forget how it goes :D